package.json

```diff
@@ -96,10 +96,11 @@
   },
   "scripts": {
     "build": "node ./bin/build.js",
+    "build-node": "node ./bin/build.js node",
     "build-as-modular-es5": "sh bin/build-as-modular-es5.sh",
     "test-unit": "mocha tests/unit/test.extras.js && npm run build-as-modular-es5 && mocha tests/unit",
-    "test-node": "sh bin/test-node.sh",
-    "test-component": "mocha tests/component",
+    "test-node": "npm run build-node && sh bin/test-node.sh",
+    "test-component": "npm run build-node && mocha tests/component",
```

Comment:
So this solves the problem of contributors not having to type `npm run build` before `npm test`, but it does an unnecessary double build on Travis